### PR TITLE
Stop using Mergeable; Replace with own native check

### PR DESCRIPTION
**Pull request by yowmamasita**
https://forto.atlassian.net/browse/ENGG-123

Because Mergeable is not working or slow at times

Why would a regex check take more than a minute finish execution?

Regex tested on all existing JIRA projects (183 projects in total)

### CODE REVIEW

File: .github/workflows/jira-copy.yml
Comment (yowmamasita) [inline .github/workflows/jira-copy.yml:1]: only used to test itself

**Review by danopia — APPROVED** [review]
I'm happy to see this happen

Would mergeable still try running on every PR without the config file?

**Comment by yowmamasita** [comment]
> Would mergeable still try running on every PR without the config file?

I think this was the _only_ config enabling Mergeable to lay its waste

**Comment by yowmamasita** [comment]
What would it take to merge this? Or should we keep Mergeable?

**Comment by danopia** [comment]
mergeable 4eva

**Pull request closed without merging**